### ISSUE TITLE
RVC Colab issue..

### ISSUE THREAD

**Issue by Valr1e** [ardha27/AI-Song-Cover-RVC#15]
Ada yg ngalamin google colab( free)selalu disconnect pas traning juga inference gak lebih dari 10min. 
Jadi sudah gak bisa lagi pakai yang free , gak tau sih yg pro .belum nyoba.
Ada gak alternative colab yg free ya??

**Comment by MinatoIsuki** [comment]
- What notebook are you using?
- What appear on the screen when you get disconnect?
- Your only alternative is to Train It Yourself (using your PC GPU and CPU).

**Comment by Valr1e** [comment]
> * What notebook are you using?
> 
>     * What appear on the screen when you get disconnect?
> 
>     * Your only alternative is to Train It Yourself (using your PC GPU and CPU).

Tidak ada notifikasi apapun hanya runtime disconnect dan pas di connect kan lagi semua file di colabnya hilang ,dan harus diinstal ulang lagi rVC dan toolsnya semua.gitu terus aja. Kalau dulu masih mending file instalasinya gak hilang pas google colabnya disconnect.

**Comment by MinatoIsuki** [comment]
I am going to reproduce this just to make sure.

Might learn Bahasa sometime, I guess :/

**Comment by MinatoIsuki** [comment]
Interesting. It just straight up give up. The training isn't even started 💀
Also, is this the screen you are getting? (Don't mind the Vietnamese language I'm using)
![image](https://github.com/ardha27/AI-Song-Cover-RVC/assets/102457568/a701abb1-9da7-4109-8718-ce24a49c1ede)

**Comment by Valr1e** [comment]
Yes, that's it. I don't understand Vietnamese. But I saw the red play button .

**Comment by simteraplications** [comment]
It seams to be only related to this particular notebook I tried a different rvc web UI one and it didn't disconnect me. How ever this one is more practicakl then the other one so would be great if it worked again.

**Comment by ardha27** [comment]
Google is now starting to banning colab thats make their server blow up. They already banning many colab that contain stable diffusion. And now they're starting to banning RVC colab. Hosting WebUI/Gradio is against colab ToS

This is what it's look like for colab that gets banned ![rn_image_picker_lib_temp_d7e8e6ca-5cfd-47dc-ab76-546bb8c8e705.jpg](https://github.com/ardha27/AI-Song-Cover-RVC/assets/48845217/f90601fa-5005-41c6-aa49-1299c8ae00d6)



This is what Chris Perry (google colab product lead) said on his twitter. ![image-6.png](https://github.com/ardha27/AI-Song-Cover-RVC/assets/48845217/c2f03beb-5686-4e98-94f4-63d763795ba0)

Fast Alternative is to join colab pro. But if you want the free alternative is to use kaggle instead of colab, and i will make the tutorial soon

**Comment by simteraplications** [comment]
Thing is I would buy google Colab but they don't allow PayPal for what ever reason so no look there.

**Comment by simteraplications** [comment]
There is already a notebook for cvc training for caggle platform but it is not in English.

**Comment by Valr1e** [comment]
Sudah coba kaggle tp gak ngerti. Tunggu tutorial aj.,,🙏

**Comment by BRLTS** [comment]
Kira-kira bisa gak sih rvc di launch dari luar colab?

Is it possible deploying the RVC without google colab? Like using VSCode?

**Comment by MinatoIsuki** [comment]
> Kira-kira bisa gak sih rvc di launch dari luar colab?
> 
> Is it possible deploying the RVC without google colab? Like using VSCode?

What do you mean of deploying the RVC?
Yes, you can use RVC without Colab but you will need special GPU with a lot of VRAM, and patience.

**Comment by BRLTS** [comment]
Is there any tutorial how to do this without colab? It doesn't matter how long the training for me

**Comment by MinatoIsuki** [comment]
> Is there any tutorial how to do this without colab? It doesn't matter how long the training for me

You can train it using your computer or Kaggle, but the Kaggle notebook is not done yet.

**Comment by BRLTS** [comment]
I'll train using my computer, where i can find the code script?

**Comment by Valr1e** [comment]
Kaggle juga error' ,😱👽 ...

"Canceled by backend. Exit code: 137"

Setelah itu langsung disconnect.
Setelah beberapa hari pake ,dan mungkin juga udah banyak yg pake kemungkinan di bann juga?💀

**Comment by simteraplications** [comment]
well adleast for you the thing opens, when I try to open this link: https://www.kaggle.com/code/ardhasemaranatha/rvc-v2-kaggle it only shows me the code but I can't run any cell.

**Comment by MinatoIsuki** [comment]
> well adleast for you the thing opens, when I try to open this link: https://www.kaggle.com/code/ardhasemaranatha/rvc-v2-kaggle it only shows me the code but I can't run any cell.

Have you followed Ardha's tutorial?
On that page, select Copy & Edit like this image and you can use Kaggle.
![image](https://github.com/ardha27/AI-Song-Cover-RVC/assets/102457568/73f5bee6-b143-491d-acc7-c85acc354228)

**Comment by MinatoIsuki** [comment]
> I'll train using my computer, where i can find the code script?

3 weeks huh?
Go to this [this repository](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI) and follow the instructions [here](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/blob/main/docs/en/README.en.md).
